data/src/main/java/com/knocklock/di/RepositoryModule.kt

```diff
@@ -0,0 +1,23 @@
+package com.knocklock.di
+
+import com.knocklock.data.repository.NotificationRepositoryImpl
+import com.knocklock.domain.repository.NotificationRepository
+import dagger.Binds
+import dagger.Module
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+
+/**
+ * @Created by 김현국 2022/12/12
+ * @Time 5:41 PM
+ */
+
+@Module
+@InstallIn(SingletonComponent::class)
+interface RepositoryModule {
+
+    @Binds
```

Comment:
```suggestion
    @Binds
     @Singleton
```

Singleton으로 하나의 객체임을 보장해주려면 Singleton 주석을 추가해줘야 해요!

Reply:
빠뜨렸네영 반영할께용~!

Reply:
위의 커밋에 같이 반영했어용 [6674ddb](https://github.com/Knock-Lock/Knock-Lock/pull/50/commits/6674ddba30e3e7869be3383c533780ecae5ea40e)